python/lsst/pipe/tasks/transformMeasurement.py

```diff
@@ -95,7 +127,7 @@ def __init__(self, measConfig, pluginRegistry, inputSchema, *args, **kwargs):
     def getSchemaCatalogs(self):
         """!Return a dict containing an empty catalog representative of this task's output."""
         transformedSrc = afwTable.BaseCatalog(self.mapper.getOutputSchema())
-        return {'transformedSrc': transformedSrc}
+        return {self.outputType: transformedSrc}
```

Comment:
The fact that you have to know `self.outputType` here bugs me, but there's nothing to be done about it now.  I have just created https://jira.lsstcorp.org/browse/DM-2850 to fix it in the future.